app/services/GraylogService.scala

```diff
@@ -48,17 +48,32 @@ class GraylogService @Inject() extends LogService {
           val responseJsonValue = JsonUtil.parseJSON(responseJson).asInstanceOf[java.util.LinkedHashMap[String, java.util.LinkedList[java.util.LinkedHashMap[String, String]]]]
           val messagesjsonValue = responseJsonValue.get("messages")
           messagesjsonValue.foreach(elem => {
-            val elemMap = elem.asInstanceOf[java.util.HashMap[String, java.util.HashMap[String, String]]]
-            val message: String = elemMap.get("message").get("message")
-            val keyIndex: Int = message.indexOf("key=")
-            keyIndex match {
-              case -1 => {
-                logs = logs :+ message
+            try {
+              val elemMap = elem.asInstanceOf[java.util.HashMap[String, java.util.HashMap[String, String]]]
+              var message: String = elemMap.get("message").get("message")
+
+              val secretKeyIndex: Int = message.indexOf("'secretKey':")
+              val secretKeyLen: Int = "'secretKey':".length()
+              if (secretKeyIndex >= 0) {
+                val prev: String = message.substring(0, secretKeyIndex)
+                val sub: String = message.substring(secretKeyIndex + secretKeyLen)
+                val secretValueIndex: Int = sub.indexOf("',")
+                val suffix: String = sub.substring(secretValueIndex + "',".length())
+                message = prev + suffix
               }
-              case _  => {
-                val filterMessage: String = message.substring(0, keyIndex)
-                logs = logs :+ filterMessage
+
+              val keyIndex: Int = message.indexOf("key=")
+              keyIndex match {
+                case -1 => {
+                  logs = logs :+ message
+                }
+                case _ => {
+                  val filterMessage: String = message.substring(0, keyIndex)
+                  logs = logs :+ filterMessage
+                }
               }
+            } catch {
+              case e: Exception => Logger.warn(s"Exception: '$message'")
```

Comment:
I'm getting a compilation error here: `not found: value message`

It looks like `var message` is defined within the `try` block (line 53), which goes out of scope before the `catch` block executes. Perhaps you meant `Logger.warn(s"Exception: '$e.getMessage()'")`? See https://docs.oracle.com/javase/8/docs/api/java/lang/Throwable.html#getMessage--

Reply:
github actions does the same, please fix @bingzhang

Reply:
there is no build error in my local. I will check the auction.

Reply:
I rerun the action, but it still gives the error.
@bodom0015 can you pull the latest pr, I run `sbt dist` in my local. it has no error.

```
[warn] warning: [options] bootstrap class path not set in conjunction with -source 1.7
[warn] 1 warning
[info] Packaging /Users/bing/github/clowder/target/scala-2.10/clowder_2.10-1.12.0-develop.jar ...
[info] Done packaging.
[info] 
[info] Your package is ready in /Users/bing/github/clowder/target/universal/clowder-1.12.0-develop.zip
[info] 
[success] Total time: 165 s, completed Oct 22, 2020 9:13:12 AM
```

Reply:
@bingzhang I cannot run it either. Try running it and pulling up the frontend. Make sure you have pushed all your commits.

Reply:
@lmarini I can run the Clowder GUI from my local.

```
[warn] play - No application found at invoker init
2020-10-22 09:24:15,100 - [INFO   ] - application - Connecting to : mongodb://127.0.0.1:27017/clowder
2020-10-22 09:24:15,281 - [WARN   ] - application - Missing mongo update add-trash-dataset. Application might be broken.
2020-10-22 09:24:15,283 - [WARN   ] - application - Missing mongo update add-trash-collection. Application might be broken.
2020-10-22 09:24:15,285 - [WARN   ] - application - Missing mongo update fixing-typehint-users. Application might be broken.
2020-10-22 09:24:15,287 - [WARN   ] - application - Missing mongo update convert-to-spaces. Application might be broken.
2020-10-22 09:24:15,288 - [WARN   ] - application - Missing mongo update fixing-taglength. Application might be broken.
```

Reply:
> @lmarini I can run the Clowder GUI from my local.
> 
> ```
> [warn] play - No application found at invoker init
> 2020-10-22 09:24:15,100 - [INFO   ] - application - Connecting to : mongodb://127.0.0.1:27017/clowder
> 2020-10-22 09:24:15,281 - [WARN   ] - application - Missing mongo update add-trash-dataset. Application might be broken.
> 2020-10-22 09:24:15,283 - [WARN   ] - application - Missing mongo update add-trash-collection. Application might be broken.
> 2020-10-22 09:24:15,285 - [WARN   ] - application - Missing mongo update fixing-typehint-users. Application might be broken.
> 2020-10-22 09:24:15,287 - [WARN   ] - application - Missing mongo update convert-to-spaces. Application might be broken.
> 2020-10-22 09:24:15,288 - [WARN   ] - application - Missing mongo update fixing-taglength. Application might be broken.
> ```

The latest commit fixes the issue.

Reply:
```
MacBook-Pro:clowder bing$ git branch
  add_graylog_settings
  add_log_settings
  branch-extractor-catalog-wireframe-viewlogs
  branch-extractor-catalog-wireframe-viewmetrics
  clean_extractors_tmpfiles
  develop
  extractor-catalog-wireframe
  fix_mismatch_extractorname
  fix_mismatch_extractorname_release_1.12.0
  improve_simple_test
  release/1.12.0
  remove_debugcode
  remove_debugcode_viewlog
* remove_secretkey
MacBook-Pro:clowder bing$ 
MacBook-Pro:clowder bing$ 
MacBook-Pro:clowder bing$ git status
On branch remove_secretkey
Untracked files:
  (use "git add <file>..." to include in what will be committed)
	scripts/tester/memoryfile
	scripts/tester/test.sh

nothing added to commit but untracked files present (use "git add" to track)
MacBook-Pro:clowder bing$ git diff
MacBook-Pro:clowder bing$ 
```

Reply:
I already committed the code.

Reply:
> > @lmarini I can run the Clowder GUI from my local.
> > ```
> > [warn] play - No application found at invoker init
> > 2020-10-22 09:24:15,100 - [INFO   ] - application - Connecting to : mongodb://127.0.0.1:27017/clowder
> > 2020-10-22 09:24:15,281 - [WARN   ] - application - Missing mongo update add-trash-dataset. Application might be broken.
> > 2020-10-22 09:24:15,283 - [WARN   ] - application - Missing mongo update add-trash-collection. Application might be broken.
> > 2020-10-22 09:24:15,285 - [WARN   ] - application - Missing mongo update fixing-typehint-users. Application might be broken.
> > 2020-10-22 09:24:15,287 - [WARN   ] - application - Missing mongo update convert-to-spaces. Application might be broken.
> > 2020-10-22 09:24:15,288 - [WARN   ] - application - Missing mongo update fixing-taglength. Application might be broken.
> > ```
> 
> The latest commit fixes the issue.

thanks for the check. that is what I mean. I rerun the action on this latest commit, but it is still failed.